pages/form.tsx

```diff
@@ -31,6 +32,7 @@ const Form = () => {
 
   return (
     <form onSubmit={onSubmit}>
+      <Modal />
```

Comment:
*THIS COMMENT is unrelated to this line*

Can you have this modal open when the "I don't know" button is clicked on the page.